lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1542,6 +1539,20 @@ def mocked_has_valid_signature(method, headers_dict, body_dict, access_key, secr
         """
         return True
 
+    def _assert_verification_approved_email(self):
+        """Check that a verification approved email was sent."""
+        self.assertEqual(len(mail.outbox), 1)
+        email = mail.outbox[0]
+        self.assertEqual(email.subject, 'Your édX ID verification was approved!')
```

Comment:
Does edX e really has an accent mark in the template? Why is that the case?

Reply:
I settings.PLATFORM_NAME  value is `édX` for tests.